lib/python/qmk/commands.py

```diff
@@ -5,6 +5,7 @@
 import platform
 import subprocess
 import shlex
```

Comment:
I'm not sure this is needed.
It's best to run `qmk pytest`, that will find things like this better than humans. :)

Reply:
Heh. Was about to post the error that occurs without the import...